services/ui-src/src/components/alerts/Alert.tsx

```diff
@@ -42,14 +46,16 @@ export const Alert = ({
             <AlertDescription>
               <Text sx={sx.descriptionText}>
                 {parseCustomHtml(description)}
-              </Text>
-              {link && (
-                <Text sx={sx.linkText}>
-                  <Link href={link} isExternal>
-                    {link}
+                {link && (
+                  <Link
+                    href={createEmailLink({ address: helpDeskEmail })}
+                    isExternal
+                  >
+                    {helpDeskEmail}
                   </Link>
-                </Text>
-              )}
+                )}
+                .
+              </Text>
```

Comment:
Blocker: This is actually changing all instances of the Alert component to have this. You can see it here: 
![Screenshot 2024-03-04 at 15 49 25](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/27705928/3d2c7582-aeac-4270-b87e-67d4f7ca6c60)

Reply:
Blocker: If we want to implement real inline links here, we'll need to pass that info through the description's HTML parser on line 48. You could modify the verbiage file to something like this: 

```
const genericErrorContent = [
  {
    type: "span",
    content:
      "Something went wrong on our end. Refresh your screen and try again.<br/>If this persists, contact the MDCT Help Desk with questions or to request technical assistance by emailing ",
  },
  {
    type: "externalLink",
    content: "mdct_help@cms.hhs.gov",
    props: {
      href: "mailto:mdct_help@cms.hhs.gov",
      target: "_blank",
    },
  },
  {
    type: "span",
    content: ".",
  },
];
```